ConcernsCaseWork/ConcernsCaseWork/Services/Cases/TestSRMAService.cs

```diff
@@ -11,7 +11,6 @@ public class TestSRMAService : ISRMAService
 	{
 		private readonly List<SRMAModel> SRMAs;
 		private readonly Random random;
-		private readonly bool autoGenerateSRMAs = false;
```

Comment:
@elielijah321 please don't remove test data generation from the test service. This makes the test service useless.